src/python/ASO/Rucio/RunTransfer.py

```diff
@@ -53,6 +54,9 @@ def algorithm(self):
         RegisterReplicas(self.transfer, self.rucioClient, self.crabRESTClient).execute()
         # do 2
         MonitorLockStatus(self.transfer, self.rucioClient, self.crabRESTClient).execute()
+        # cleanup
+        # For now, only delete files in temp RSE
```

Comment:
Why "for now" ? What more shall we do ? Do you mean we delete the file but do not remove the replica ? If that, better state it clearly (and open an issue, so we do not forget)

Reply:
I have removed line 58 because it is an implementation detail that belongs to the Cleanup.py files. 
Will open the new issue about this soon.